package com.jv.crud_operation.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import org.springframework.web.servlet.view.RedirectView;

@RestController
@RequestMapping("/")
public class IndexController {
	
	@GetMapping({"/api-doc","/api-docs"})
	public RedirectView apiDoc() {
		return new RedirectView("/swagger-ui/index.html");
	}
}
